app/javascript/techradar.js

```diff
@@ -55,76 +63,131 @@
                     });
 
                 // creates all spots
-                createRadarSpots(enterElement);
+                createRadarSpots(enterElement, quadrant);
 
                 // write a head-line per quadrant
                 createListHeadline(quadrant, (quadrantIndex < 2) ? 0 : 1);
                 createSpotLabelAndListElements(enterElement, quadrant);
             }, this);
 
             renderHelp(canvas, this.radar);
-            
-            function sortByArcOrder(a,b) {
+            renderTooltip(canvas);
```

Comment:
Per spot basis rendering would create a lot of content that is not visible at the same time. Tooltip inherently is something that is visible only for one spot - so I had one element that is repositioned and shown/hidden. This also removes the risk of having multiple tooltips visible at once.